docs/references/nextjs/auth-middleware.mdx

```diff
@@ -40,15 +67,52 @@ export default authMiddleware({
     if (!auth.userId && !auth.isPublicRoute) {
       return redirectToSignIn({ returnBackUrl: req.url });
     }
-    // redirect them to organization selection page
+    // redirect logged in users to organization selection page if they are not active in an organization
     if(auth.userId && !auth.orgId && req.nextUrl.pathname !== "/org-selection"){
       const orgSelection = new URL('/org-selection', req.url)
       return NextResponse.redirect(orgSelection)
     }
+    // if the user is logged in and trying to access a protected route, allow them to access route
+    if (auth.userId && !auth.isPublicRoute) {
+      return NextResponse.next()
+    }
+    // allow users visiting public routes to access them
+    return NextResponse.next();
   }
 });
 ```
 
+#### Understanding `afterAuth` configuration
+
+When using `afterAuth`, the checks must handle all cases—a custom `afterAuth` completely overrides the default behavior covered earlier aside from adding sign-in and sign-up to `publicRoutes`. `afterAuth` does not allow you to add just one check and rely on the default behavior otherwise. 
```

Comment:
```suggestion
When using `afterAuth`, the checks must handle all cases — a custom `afterAuth` completely overrides the default behavior covered earlier, aside from adding sign-in and sign-up to `publicRoutes`. `afterAuth` does not allow you to add just one check and rely on the default behavior otherwise. 
```